helpers.go

```diff
@@ -0,0 +1,37 @@
+package main
+
```

Comment:
everything in this file seems like it could be moved to `excel.go` or `requests.go`

Reply:
Done.